_releases/29.0.md

```diff
@@ -167,7 +167,7 @@ The build system has been migrated from Autotools to CMake:
 
 1. The minimum required CMake version is 3.22.
 2. In-source builds are not allowed. When using a subdirectory within the root source tree as a build directory, it is recommended that its name includes the substring "build".
-3. CMake variables may be used to configure the build system. See [Autotools to CMake Options Mapping](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/Autotools-to-CMake-Options-Mapping) for details.
+3. CMake variables may be used to configure the build system. **Some defaults have changed.** For example, you will now need to add `-WITH_ZMQ=ON` to build with zmq and `-BUILD_GUI=ON` to build `bitcoin-qt`. See [Autotools to CMake Options Mapping](https://github.com/bitcoin-core/bitcoin-devwiki/wiki/Autotools-to-CMake-Options-Mapping) for details.
```

Comment:
if you add the `-` to write the command and not just the variable name, should also add the `D`: `-DWITH_ZMQ=ON` and `-DBUILD_GUI=ON`

Reply:
changed